tests/e2e/resources/installation_config/cognito.yaml

```diff
@@ -0,0 +1,272 @@
+    helm: "../../charts/apps/admission-webhook"
+  validations:
+    namespace: 'kubeflow'
+    identifier: 'app'
+    common_label: 'poddefaults'
+
+#jupyter-web-app
+jupyter-web-app:
+  installation_options:
+    kustomize: 
+      - "../../awsconfigs/apps/jupyter-web-app"
+    helm: "../../charts/apps/jupyter-web-app"
+  validations:
+    namespace: 'kubeflow'
+    identifier: 'app'
+    common_label: 'jupyter-web-app'
+
+#notebook-controller
+notebook-controller:
+  installation_options:
+    kustomize: 
+      - "../../upstream/apps/jupyter/notebook-controller/upstream/overlays/kubeflow"
+    helm: "../../charts/apps/notebook-controller"
+  validations:
+    namespace: 'kubeflow'
+    identifier: 'app'
+    common_label: 'notebook-controller'
+
+#volume-web-app
+volumes-web-app:
+  installation_options:
+    kustomize: 
+      - "../../upstream/apps/volumes-web-app/upstream/overlays/istio"
+    helm: "../../charts/apps/volumes-web-app"
+  validations: 
+    namespace: 'kubeflow'
+    identifier: 'app'
+    common_label: 'volumes-web-app'
+
+#training-operator
+training-operator:
+  installation_options:
+    kustomize: 
+      - "../../upstream/apps/training-operator/upstream/overlays/kubeflow"
+    helm: "../../charts/apps/training-operator"
+  validations:
+    namespace: 'kubeflow'
+    identifier: 'control-plane'
+    common_label: 'kubeflow-training-operator'
+
+#katib
+katib:
+  installation_options:
+    kustomize: 
+      - "../../upstream/apps/katib/upstream/installs/katib-with-kubeflow"
+    helm: "../../charts/apps/katib/vanilla"
+  validations:
+    namespace: 'kubeflow'
+    identifier: 'katib.kubeflow.org/component'
+    common_label: 'controller, db-manager, ui'
+
+#tensorboard
+tensorboard-controller:
+  installation_options:
+    kustomize: 
+      - "../../upstream/apps/tensorboard/tensorboard-controller/upstream/overlays/kubeflow"
+    helm: "../../charts/apps/tensorboard-controller"
+  validations:
+    namespace: 'kubeflow'
+    identifier: 'app'
+    common_label: 'tensorboard-controller'
+
+tensorboards-web-app:
+  installation_options:
+    kustomize:  
+      - "../../upstream/apps/tensorboard/tensorboards-web-app/upstream/overlays/istio"
+    helm: "../../charts/apps/tensorboards-web-app"
+  validations:
+    namespace: 'kubeflow'
+    identifier: 'app'
+    common_label: 'tensorboards-web-app'
+
+#profiles and kfam
+profiles-and-kfam:
+  installation_options:
+    kustomize: 
+      - "../../upstream/apps/profiles/upstream/overlays/kubeflow"
+    helm: "../../charts/apps/profiles-and-kfam"
+  validations:
+    namespace: 'kubeflow'
+    identifier: 'kustomize.component'
+    common_label: 'profiles'
+
+#Ingress
+ingress:
+  installation_options:
+    kustomize: 
+      - "../../awsconfigs/common/istio-ingress/overlays/cognito"
+    helm: "../../charts/common/ingress/cognito"
+
+
+#ALB Controller
+aws-load-balancer-controller:
+  installation_options:
```

Comment:
should we do:
```
kustomize:
  repo: local
  paths:
    - ./path
    - ./path
helm:
  repo: remote
```    
assigning keys to values gives flexibility to add more configurations in future without breaking compatiblity

Reply:
ok. will add the repo key to the dict

Reply:
so for helm it will be just 
```
helm:
   repo:
   path:
```
since I don;t think it can have multiple paths